### PR TITLE
add FT to cover the AC

### PR DESCRIPTION
**Pull request by iguannago**
https://tools.hmcts.net/jira/browse/RWA-595

### CODE REVIEW

File: src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskForSearchCompletionControllerTest.java [hunk 1]
```diff
@@ -50,6 +50,37 @@ public void setUp() {
...
+            ENDPOINT_BEING_TESTED,
+            decideAnApplicationSearchRequest,
+            authenticationHeaders
+        );
+
+        result.then().assertThat()
+            .statusCode(HttpStatus.OK.value())
+            .contentType(APPLICATION_JSON_VALUE)
+            .body("tasks.size()", equalTo(1));
+
```
Comment (indvamshi) [inline src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskForSearchCompletionControllerTest.java:80]: if possible can you verify the task type.

Comment (iguannago) [reply src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskForSearchCompletionControllerTest.java:81]: done. Thanks

**Review by indvamshi — APPROVED** [review]

**Review by alexbaeza — APPROVED** [review]

File: src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskForSearchCompletionControllerTest.java [hunk 2]
```diff
@@ -50,6 +50,38 @@ public void setUp() {
...
+            decideAnApplicationSearchRequest,
+            authenticationHeaders
+        );
+
+        result.then().assertThat()
+            .statusCode(HttpStatus.OK.value())
+            .contentType(APPLICATION_JSON_VALUE)
+            .body("tasks.size()", equalTo(1))
+            .body("tasks[0].type", equalTo("processApplication"));
+
```
Comment (BenNzewi) [inline src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskForSearchCompletionControllerTest.java:81]: Can we add the test to capture the scenario where there is no match and an empty list is returned?

**Comment by iguannago** [comment]
moved here: https://github.com/hmcts/wa-task-management-api/pull/259/files
